README.md

```diff
@@ -0,0 +1,4 @@
+# curve-stablecoin
+Stablecoin powered by LLAMMAs
+
+Protocol description in [doc/curve-stablecoin.pdf](https://github.com/dcbuild3r/curve-stablecoin/blob/master/doc/curve-stablecoin.pdf)
```

Comment:
Probably don't want to link to a forked repo :wink: 

```suggestion
Protocol description in [doc/curve-stablecoin.pdf](./doc/curve-stablecoin.pdf)
```

Reply:
my bad, didn't notice when copypasting